src/ErrorBoundaries/ErrorFallbackComponent.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import { SafeAreaView, View, StyleSheet } from "react-native"
+import { useTranslation } from "react-i18next"
+import { Button } from "../components/Button"
+import { RTLEnabledText } from "../components/RTLEnabledText"
+
+import { Colors, Spacing, Typography } from "../styles"
+
+type Props = { error: Error | string; resetError: () => void }
```

Comment:
an interface might be a better fit here, or a least thats the convention thats been established in other components.